src/fraud.rs

```diff
@@ -0,0 +1,38 @@
+use jsonrpsee::proc_macros::rpc;
+use jsonrpsee::core::{Error};
+use protobuf::Message;
+use crate::proto::{byzantine};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum ProofType {
+    BadEncoding
```

Comment:
```suggestion
    BadEncoding,
```

see the naming conventions for example [here](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter)